lib/DDG/Spice/SublimePackages.pm

```diff
@@ -0,0 +1,28 @@
+package DDG::Spice::SublimePackages;
+# ABSTRACT: Search for information about Sublime Text packages
+
+use strict;
+use DDG::Spice;
+
+name "SublimePackages";
+source "packagecontrol.io";
+icon_url "/i/packagecontrol.io.ico";
+description "Display Sublime Text packages";
+primary_example_queries "sublimetext package code", "sublime text package php";
+category "software";
+topics "programming", "computing";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/SublimePackages.pm";
+attribution github => ["MrChrisW", "Chris Wilson"],
+            web => ["http://chrisjwilson.com", "Chris Wilson"];
+
+spice to => 'https://packagecontrol.io/search/$1.json';
+spice wrap_jsonp_callback => 1;
+
+triggers startend => "sublimetext package", "sublime text package", "sublime text packages";
```

Comment:
@jagtalon Add "sublime text package manager" as a full trigger e.g 
`sublime text package manager <keyword>`

>Also I see myself just typing in "sublime text perl". See if you might need to use DDG.isRelevant or the relevancy block for that.

Sure, I was keeping the triggers limited to start with, happy to add sublime text as a trigger and ensure relevancy.